api/public/v2/tests/test_api_component_viewset.py

```diff
@@ -59,6 +89,6 @@ def test_component_list(self, commit_compontents, get_repo_permissions):
         res = self._request_components()
         assert res.status_code == 200
         assert res.json() == [
-            {"component_id": "foo", "name": "Foo"},
-            {"component_id": "bar", "name": "Bar"},
+            {"component_id": "foo", "name": "Foo", "coverage": 62.5},
```

Comment:
might be helpful to add a comment on the sample report doing the math on the coverage calc for someone looking at this later

Took me a minute to confirm the values 😅